### PR TITLE
Caitlin's slides.

### PR DESCRIPTION
**Pull request by csteinert**

### CODE REVIEW

File: netjets-user-group/.gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+.DS_Store
```
Comment (mewdriller) [inline netjets-user-group/.gitignore]: The dreaded `.DS_Store`!

File: netjets-user-group/slides.md [hunk 2]
```diff
@@ -28,29 +28,118 @@ Presented by:
...
+----
+
+![Image of HTML5 and CSS3 logos](http://www.bobbyberberyan.com/wp-content/uploads/2012/03/HTML5CSS3Logos.svg)
+
+Note:
+...promoting and facilitating the use of the latest and greatest features in HTML and CSS...
+
+----
+
+####Less of this
```
Comment (mewdriller) [inline netjets-user-group/slides.md]: It seems like it doesn't mind it, but I think _technically_ markdown asks for a space between the `####` and the `h4` text.

File: netjets-user-group/slides.md [hunk 3]
```diff
@@ -28,29 +28,118 @@ Presented by:
...
+
+![Image of machine mass producing crayons](https://i.kinja-img.com/gawker-media/image/upload/s--e8JjQnyR--/c_fit,fl_progressive,q_80,w_636/19dg5vejfp529gif.gif)
+
+Note:
+...reducing the amount of repeated work required as apps begin to switch over to the new design...
+
+----
+
+####More of this
```
Comment (mewdriller) [inline netjets-user-group/slides.md]: Ditto about the space after the `####`.

File: netjets-user-group/slides.md [hunk 4]
```diff
@@ -28,29 +28,118 @@ Presented by:
...
+- As they create guidelines, we are simultaneously translating them into the Living Styleguide
+
+----
+
+![Image of styleguide button](images/button.png)
+
+Note:
+- This is a screenshot of the Living Styleguide that shows the default styles for a basic button element
+
+
```
Comment (mewdriller) [inline netjets-user-group/slides.md]: Extra newline here.

File: netjets-user-group/slides.md [hunk 5]
```diff
@@ -28,29 +28,118 @@ Presented by:
...
+![Image of HTML5 and CSS3 logos](http://www.bobbyberberyan.com/wp-content/uploads/2012/03/HTML5CSS3Logos.svg)
+
+Note:
+...promoting and facilitating the use of the latest and greatest features in HTML and CSS...
+
+----
+
+####Less of this
+
+![Image of machine mass producing crayons](https://i.kinja-img.com/gawker-media/image/upload/s--e8JjQnyR--/c_fit,fl_progressive,q_80,w_636/19dg5vejfp529gif.gif)
```
Comment (mewdriller) [inline netjets-user-group/slides.md]: Should we pull this image local?

File: netjets-user-group/slides.md [hunk 6]
```diff
@@ -28,29 +28,118 @@ Presented by:
...
+----
+
+![Image of styleguide flight pair](images/flight-pair.png)
+
+Note:
+...creating markup and styles that are custom-tailored to the NetJets brand and apps...
+
+----
+
+![Image of HTML5 and CSS3 logos](http://www.bobbyberberyan.com/wp-content/uploads/2012/03/HTML5CSS3Logos.svg)
```
Comment (mewdriller) [inline netjets-user-group/slides.md]: This image is also remote; maybe pull this in?

File: netjets-user-group/slides.md [hunk 7]
```diff
@@ -28,29 +28,118 @@ Presented by:
...
+![Image of machine mass producing crayons](https://i.kinja-img.com/gawker-media/image/upload/s--e8JjQnyR--/c_fit,fl_progressive,q_80,w_636/19dg5vejfp529gif.gif)
+
+Note:
+...reducing the amount of repeated work required as apps begin to switch over to the new design...
+
+----
+
+####More of this
+
+![Image of a pit crew](https://media.giphy.com/media/MAQR6SBvKGQ5a/giphy.gif)
```
Comment (mewdriller) [inline netjets-user-group/slides.md]: Another remote image.

File: netjets-user-group/slides.md [hunk 8]
```diff
@@ -28,29 +28,118 @@ Presented by:
...
+![Image of styleguide inventory list](images/inventory-list.png)
+
+Note:
+- ...which becomes especially useful as we group elements into more complex components
+
+----
+
+## Living Styleguide
+
+ - Creates a common design language
```
Comment (mewdriller) [inline netjets-user-group/slides.md]: So it absolutely doesn't matter, but just as an FYI: you don't need the ` ` before the `-` for a bulleted list (or a numbered one). I just recently found that out and I felt like I liked it better without. ¯\_(ツ)_/¯

**Comment by bradpearson** [comment]
+1

**Comment by mewdriller** [comment]
Awesome stuff @csteinert! :+1:

**Comment by bradpearson** [comment]
+1

**Pull request merged**